docs/ci-cd-environment/agent-migration-to-ubuntu2004.md

```diff
@@ -0,0 +1,42 @@
+---
+description: This guide specifies the steps to change Semaphore 2.0 agent OS from Ubuntu 18.04 to Ubuntu 20.04
+---
+
+# Migration from Ubuntu 18.04 image to Ubuntu 20.04 image 
+
+If you've used Semaphore Ubuntu 18.04 image, this guide will outline the
+key differences and provide you with a direction to migrate to the Ubuntu 20.04 image.
+
+### You don't have to migrate
+
+We know how much you rely on Semaphore to do your work and don't want to impose
+migration timeline. We won't deprecate the Ubuntu 18.04 image.
+
+### Benefits of using the Ubuntu 20.04
+
+Ubuntu 20.04 LTS (Focal Fossa) is the latest lts release for the Ubuntu linux
+server operating system. 
+Compared to the previous LTS version, this release has enhanced resource usage and includes a newer kernel.
+Canonical will support Ubuntu 20.04 until April 2025. 
+If you want to read more about Ubuntu 20.04 check out (Ubuntu Blog)[https://ubuntu.com/blog/ubuntu-server-20-04]
+
+### Ubuntu 20.04 image on Semaphore 2.0
```

Comment:
IIRC we resolved not to write '2.0' in any new content but if there's something to dispute me go ahead.